contract/src/constants.rs

```diff
@@ -27,6 +27,9 @@ pub const ARG_NFT_PACKAGE_HASH: &str = "cep78_package_key";
 pub const ARG_NAMED_KEY_CONVENTION: &str = "named_key_convention";
 pub const ARG_ACCESS_KEY_NAME_1_0_0: &str = "access_key_name";
 pub const ARG_HASH_KEY_NAME_1_0_0: &str = "hash_key_name";
+pub const ARG_ADDITIONAL_REQUIRED_METADATA: &str = "additional_required_metadata";
+pub const ARG_OPTIONAL_METADATA: &str = "optional_metadata";
+
```

Comment:
Unneeded white line?

Reply:
The ARG and ENTRY_POINT parts in constants.rs have been separated since a previous PR

Reply:
Cool